server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -94,6 +98,47 @@ function selectProtocolVersion(connectVersions: string[]): string {
     }
 }
 
+/**
+ * @returns true if throttled and sent nack; false if not throttled or no throttler provided.
+ */
```

Comment:
sends

Reply:
Maybe I could rephrase this better, but past tense was intentional because it will return true _after_ it has "sent" a nack. Perhaps something like
```ts
/**
 * When throttled, sends nack before returning.
 *
 * @returns true if throttled; false if not throttled or no throttler provided.
 */
```

Reply:
Yeah, that looks good.